cmd/state/internal/cmdtree/cve.go

```diff
@@ -25,3 +26,29 @@ func newCveCommand(prime *primer.Values) *captain.Command {
 	cmd.SetGroup(PlatformGroup)
 	return cmd
 }
+
+func newReportCommand(prime *primer.Values) *captain.Command {
+	report := cve.NewReport(prime)
+	params := cve.ReportParams{
+		Namespace: &project.Namespaced{},
```

Comment:
Why initialize an empty namespace? Shouldn't this be nil if it wasn't provided?

Reply:
This is how we did in [other places](https://github.com/ActiveState/cli/blob/master/cmd%2Fstate%2Finternal%2Fcmdtree%2Factivate.go#L19-L20) to set the namespace as an argument, by calling its `Set()` method as required by the `ArgMarshaler` interface.